### PR TITLE
Modified comments in 4_switch_controlled_motor_normal

### PR DESCRIPTION
**Pull request by PritK99**
Modified comments in 4_switch_controlled_motor_normal

### CODE REVIEW

**Comment by Jamm02** [comment]
@RISHI27-dot please review the PR

File: 4_switch_controlled_motor_normal/main/switch_control_motor_normal.c [hunk 1]
```diff
@@ -19,62 +19,70 @@ void drive_task(void *arg)
...
+			 * runs the second motor A_1 for 80 duty cycles backwards 
+			 * more details can be found at https://github.com/SRA-VJTI/sra-board-component/blob/9f28700759ac816660c18859d65303d6540e8732/src/motor_driver.c#L195
+			 */
 			set_motor_speed(MOTOR_A_1, MOTOR_BACKWARD, 80);
 			ESP_LOGI(TAG_SWITCH_CONTROL_MOTOR, "MOTOR_A_1 BACKWARD");
 		}
 		else
 		{
-			//if there is no input from LSA for the second motor then stop the motor
+			// if neither of switch 3 or switch 4 is pressed on then stop the motor A_1
```
Comment (RISHI27-dot) [inline 4_switch_controlled_motor_normal/main/switch_control_motor_normal.c:85]: Change to "  if neither of switch 3 or switch 4 is pressed, stop the motor A_1. "

File: 4_switch_controlled_motor_normal/main/switch_control_motor_normal.c [hunk 2]
```diff
@@ -19,62 +19,70 @@ void drive_task(void *arg)
...
+			 * runs the first motor A_0 for 80 duty cycles backwards
+			 * more details can be found at https://github.com/SRA-VJTI/sra-board-component/blob/9f28700759ac816660c18859d65303d6540e8732/src/motor_driver.c#L195
+			 */
 			set_motor_speed(MOTOR_A_0, MOTOR_BACKWARD, 80);
 			ESP_LOGI(TAG_SWITCH_CONTROL_MOTOR, "MOTOR_A_0 BACKWARD");
 		}
 		else
 		{
-			// if there is no input from  LSA for first motor then stop the motor, this stops the motor if the previous 80 duty cycles are not completed
+			// if neither of switch 1 or switch 2 is pressed on then stop the motor A_0
```
Comment (RISHI27-dot) [inline 4_switch_controlled_motor_normal/main/switch_control_motor_normal.c:59]: Change to " if neither of switch 1 or switch 2 is pressed, stop the motor A_0. "

File: 4_switch_controlled_motor_normal/README.md [hunk 3]
```diff
@@ -22,7 +22,14 @@
...
 - ### **Working**
-    - Activates GPIO swtiches to take input from LSA
-    - Activates one of the two TB6612FNG drivers
-    - takes input from LSA  
-    - runs the bot motors in Normal mode according to the input.
+    - Activates GPIO pins for swtiches 
+    - Activates the TB6612FNG driver
+    - runs the bot motors in Normal mode and controls the motors according to the switches activated.
+    <br></br>
+    The following flowchart represents the flow of code
```
Comment (RISHI27-dot) [inline 4_switch_controlled_motor_normal/README.md:29]: Change to " The code flow is illustrated in the flowchart below. "

File: 4_switch_controlled_motor_normal/README.md [hunk 4]
```diff
@@ -22,7 +22,14 @@
         <br/>
 
 - ### **Working**
-    - Activates GPIO swtiches to take input from LSA
-    - Activates one of the two TB6612FNG drivers
-    - takes input from LSA  
-    - runs the bot motors in Normal mode according to the input.
+    - Activates GPIO pins for swtiches 
+    - Activates the TB6612FNG driver
+    - runs the bot motors in Normal mode and controls the motors according to the switches activated.
```
Comment (RISHI27-dot) [inline 4_switch_controlled_motor_normal/README.md:27]: runs -> Runs.
Normal -> normal

**Review by RISHI27-dot — CHANGES_REQUESTED** [review]

**Pull request closed without merging**